src/signal.rs

```diff
@@ -1058,6 +1068,15 @@ mod combobox {
         callback_guard!();
         f(&ComboBox::from_glib_none(this).downcast_unchecked(), scroll_type);
     }
+
+    unsafe extern "C" fn format_entry_text_trampoline<T>(
+        this: *mut GtkComboBox,
+        path: *mut c_char, f: &Box<Fn(&T, &str) -> String + 'static>) -> *mut c_char
+    where T: IsA<ComboBox> {
```

Comment:
I never remember how @gkoz wants where indentation. Let's wait for his advice. :)